library/common/config/templates.h

```diff
@@ -77,6 +77,26 @@ extern const char* brotli_config_insert;
  */
 extern const char* socket_tag_config_insert;
 
+/* Insert for default layers.
+ */
+extern const char* default_layers_insert;
+
+/* Insert that enables default stats config.
+ */
+extern const char* default_stats_config_insert;
+
+/* Insert that enables rtds clusters.
```

Comment:
I think this should be xDS clusters.... there might be one or more xDS clusters and the various xDS APIs might make use of one or more of those xDS clusters... a cluster may not be specific to just rtds or sds etc